modules/EvseManager/evse/evse_managerImpl.cpp

```diff
@@ -60,6 +60,16 @@ void evse_managerImpl::init() {
     mod->mqtt.subscribe(fmt::format("everest_external/nodered/{}/cmd/resume_charging", mod->config.connector_id),
                         [&charger = mod->charger](const std::string& data) { charger->resumeCharging(); });
 
+    mod->mqtt.subscribe(fmt::format("everest_external/nodered/{}/cmd/stop_transaction", mod->config.connector_id),
```

Comment:
the fmt::format should be a framework helper function which automatically adds the current correct external prefix